src/lib/viewers/controls/controls-layer/ControlsLayer.tsx

```diff
@@ -74,12 +75,21 @@ export default function ControlsLayer({ children, onMount = noop }: Props): JSX.
         onMount(helpersRef.current);
     }, [onMount]);
 
+    // Hides control layer when forceShow changes to false
+    React.useEffect(() => {
+        helpersRef.current.reset();
+
+        if (!forceShow) {
+            helpersRef.current.hide();
+        }
+    }, [forceShow]);
+
     // Destroy timeouts on unmount
     React.useEffect(() => helpersRef.current.clean, []);
 
     return (
         <div
-            className={`bp-ControlsLayer ${isShown ? SHOW_CLASSNAME : ''}`}
+            className={`bp-ControlsLayer ${isShown || forceShow ? SHOW_CLASSNAME : ''}`}
```

Comment:
Does `forceShow` need to be maintained separately from `isShown` state? Or can it be rolled into the state value?